lib/digest/crc5.rb

```diff
@@ -64,7 +53,7 @@ def self.pack(crc)
     #
     def update(data)
       data.each_byte do |b|
-        @crc = ((@table[((@crc >> 3) ^ b) & 0xff] ^ (@crc >> 8)) & @crc_mask)
+        @crc = @table[(@crc ^ b) & 0xff] ^ (@crc >> 8)
```

Comment:
The C code which [pycrc-0.10.0](https://pycrc.org/) (reference implementation) generated still uses a CRC mask of `0x1f`.

```shell
/src/pycrc.py --generate c -o crc5.c --model crc-5 --algorithm=table-driven
```

```c
crc_t crc_update(crc_t crc, const void *data, size_t data_len)
{
    const unsigned char *d = (const unsigned char *)data;
    unsigned int tbl_idx;

    while (data_len--) {
        tbl_idx = crc ^ *d;
        crc = (crc_table[tbl_idx] ^ (crc >> 8)) & 0x1f;
        d++;
    }
    return crc & 0x1f;
}
```

I think the `& @crc_mask` should remain.